src/app/workspace/resource/representation/still-image/still-image.component.html

```diff
@@ -1,8 +1,16 @@
 <!-- openseadragon (osd) viewer -->
 <div class="osd-container" [class.drawing]="regionDrawMode">
     <div class="action horizontal top overlay">
+        <div class="captions-container">
+            <div class="top">
+                <span class="caption mat-caption">{{imageCaption}}</span>
+            </div>
+            <div class="bottom">
+                <span class="iiif mat-caption">{{iiifUrl}}</span>
```

Comment:
it would be nice to define it as a link to open the image only in a new tab.

Reply:
@kilchenmann I had the same thought! Added in 5f1cf90 😉